_config.yml

```diff
@@ -10,4 +10,5 @@ permalink: :title/
 relative_permalinks: false
 
 api: "https://api.carpoolvote.com/live"
-cp_site: "http://carpoolvote.com"
\ No newline at end of file
+cp_site: "http://carpoolvote.com"
+exlcude: ['node_modules']
```

Comment:
exclude is misspelled here.

Reply:
@dlamoureaux great, thank you for reviewing this PR and finding this mistake 👍 I've corrected this now